### PR TITLE
QDQ tool modification part3

### PR DESCRIPTION
**Pull request by chilo-ms**
Refine code for support of QDQ per channel quantization with channel axis specified.

Remove:

- OpTypesSupportPerChannelQuantization, QDQChannelAxis 

Add:

- QDQOpTypePerChannelSupportToAxis = dictionary : Default is {}. Set channel axis for specific op type, for example: {'MatMul': 1}, and it's effective only when per channel quantization is supported and per_channel is True. If specific op type supports per channel quantization but not explicitly specified with channel axis, default channel axis will be used.

### CODE REVIEW

File: onnxruntime/python/tools/quantization/operators/matmul.py [hunk 1]
```diff
@@ -98,3 +99,24 @@ def quantize(self):
...
+        super().__init__(onnx_quantizer, onnx_node)
+
+    def quantize(self):
+        node = self.node
+        assert (node.op_type == "MatMul")
+
+        if self.disable_qdq_for_node_output:
+            nodes_to_iterate = node.input
+        else:
+            nodes_to_iterate = itertools.chain(node.input, node.output)
```
Comment (jywu-msft) [inline onnxruntime/python/tools/quantization/operators/matmul.py:114]: missing module import for itertools

Comment (chilo-ms) [reply onnxruntime/python/tools/quantization/operators/matmul.py:115]: itertools module added

**Review by jywu-msft — APPROVED** [review]

**Comment by stevenlix** [comment]
Please also update https://github.com/microsoft/onnxruntime-inference-examples/pull/56

**Comment by chilo-ms** [comment]
> Please also update [microsoft/onnxruntime-inference-examples#56](https://github.com/microsoft/onnxruntime-inference-examples/pull/56)

Will do

**Review by yufenglee — APPROVED** [review]

**Pull request merged**